Source/i_system.c

```diff
@@ -285,6 +286,7 @@ void I_Init(void)
   // killough 3/6/98: end of keyboard / autorun state changes
 
    I_AtExit(I_Shutdown, true);
+   I_AtExitPrio(I_ErrorMsg, true, "I_ErrorMsg", exit_priority_verylast);
```

Comment:
Should I provide a "very very last" category, or simply apply sequence numbers, or just keep it? Currently, `Z_Close()` is executed one step later than this.

Reply:
I think it's ok to keep it as is. The exit sequence already seems complicated.